src/features/animation/components/animation-tabs.tsx

```
@@ -0,0 +1,103 @@
+                  {tab.saved ? <i className="ri-bookmark-fill" /> : null}
+                  {tab.title}
+                </span>
+                {tabs.length > 1 && !presentational ? (
+                  <span
+                    role="button"
+                    className="-mr-2.5 ml-1 transition-opacity opacity-0 group-hover/tab:opacity-100 inline-flex items-center justify-center w-4 h-4 rounded-md hover:bg-accent hover:text-accent-foreground cursor-pointer"
+                    onClick={(e) => {
+                      e.stopPropagation();
+                      handleRemoveTab(tab.id);
+                    }}
+                  >
```

Biome (2.1.2)

[error] The HTML element with the interactive role "button" is not focusable.

A non-interactive HTML element that is not focusable may not be reachable for users that rely on keyboard navigation, even with an added role like "button".
Add a tabIndex attribute to make this element focusable.

(lint/a11y/useFocusableInteractive)